### PR TITLE
zh, en: add some description about configurations

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added, or deleted? (Required)

align with https://github.com/pingcap/dm/pull/1817

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/source-configuration-file.md [hunk 1]
```diff
@@ -85,11 +90,21 @@ from:
...
 
+### 任务状态检查配置（checker 配置项）
+
+DM 会定期检查根据任务状态以及错误信息，判断恢复任务能否消除错误，并自动尝试恢复任务进行重试。DM 会使用指数回退策略调整检查间隔。这些行为可以通过如下配置进行调整：
+
+| 配置项        | 说明                                    |
+| :------------ | :--------------------------------------- |
+| `check-enable` | 启用自动重试功能。 |
+| `backoff-rollback` | 如果指数回退策略的间隔大于该值，且任务处于正常状态，尝试减小间隔。 |
+| `backoff-rollback` | 指数回退策略的间隔的最大值，该值必须大于 1 秒。 |
```
Comment (GMHDBJD) [inline zh/source-configuration-file.md]: backoff-max

**Review by GMHDBJD — COMMENTED** [review]
And we should add checker config for English doc.

**Comment by GMHDBJD** [comment]
> And we should add checker config for English doc.

Seems already exist. 😄

**Review by GMHDBJD — APPROVED** [review]
LGTM

File: en/source-configuration-file.md [hunk 2]
```diff
@@ -40,6 +40,11 @@ from:
 #   expires: 0
 #   remain-space: 15
 
+# checker:
+#   check-enable: true
+#   backoff-rollback: 5m0s
+#   backoff-max: 5m0s       # The max value of backoff, should be larger than 1s
```
Comment (Liuxiaozhen12) [inline en/source-configuration-file.md:46]:
```suggestion
#   backoff-max: 5m0s       # The maximum value of backoff, should be larger than 1s
```

File: en/source-configuration-file.md [hunk 3]
```diff
@@ -89,6 +94,16 @@ Generally, there is no need to manually configure these parameters unless there
 >
 > The automatic data purge strategy only takes effect when `interval` is not 0 and at least one of the two configuration items `expires` and `remain-space` is not 0.
 
+### Task status checker configuration (`checker`)
+
+DM periodically checks the current task status and error message, to determine if resuming the task will eliminate the error, and automatically retries to resume the task if needed. DM adjusts the checking interval using an exponential backoff strategy. These behaviors can be adjusted by the following configuration.
```
Comment (Liuxiaozhen12) [inline en/source-configuration-file.md:99]:
```suggestion
DM periodically checks the current task status and error message to determine if resuming the task will eliminate the error. If needed, DM automatically retries to resume the task. DM adjusts the checking interval using the exponential backoff strategy. Its behaviors can be adjusted by the following configuration.
```

File: en/source-configuration-file.md [hunk 4]
```diff
@@ -89,6 +94,16 @@ Generally, there is no need to manually configure these parameters unless there
...
+
+| Parameter        | Description                                    |
+| :------------ | :--------------------------------------- |
+| `check-enable` | Whether enables this feature. |
```
Comment (Liuxiaozhen12) [inline en/source-configuration-file.md:103]:
```suggestion
| `check-enable` | Whether to enable this feature. |
```

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**